src/client/localplayer.cpp

```diff
@@ -433,12 +433,14 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 	}
 
 	/*
-		Check properties of the node on which the player is standing
+		Check properties of the node on which the player is standing or in which the player's feet
 	*/
+	v3s16 pos = getStandingNodePos();
 	const ContentFeatures &f = nodemgr->get(map->getNodeNoEx(m_standing_node));
+	const ContentFeatures &f1 = nodemgr->get(map->getNodeNoEx(pos + v3s16(0, 1, 0)));
 
 	// Determine if jumping is possible
-	m_disable_jump = itemgroup_get(f.groups, "disable_jump");
+	m_disable_jump = (itemgroup_get(f.groups, "disable_jump") or itemgroup_get(f1.groups, "disable_jump"));
```

Comment:
Outer commas are not needed.
Please check the line is no longer than 90 columns (with tab size 4).

Reply:
@MoNTE48 Also, why is `or` used in a C++ file? Shouldn't it be `||`?

Reply:
Hehe 'commas' =)